packages/zero-unplugin/src/index.ts

```diff
@@ -250,8 +244,10 @@ export const plugin = createUnplugin<PluginOptions, true>((options) => {
     },
   };
 
-  const plugins: Array<UnpluginOptions> = [
-    {
+  const plugins: Array<UnpluginOptions> = [];
+
+  if (theme) {
```

Comment:
You'll need to do the same thing for vite plugin. Also, why do we need this?
If we are adding the plugin conditionally, we'll also need to modify the styled processor to not add the 3rd argument in the doRuntimeReplacement method.

Reply:
Got it. I switched it back and return `null` for non-nextjs instead. Please review it again.